src/components/Map.vue

```diff
@@ -221,6 +221,8 @@ export default {
 
     showNearbyStations: function(selectedStation) {
       const stationNumber = selectedStation.number;
+      //get all detectives to check locations
+      const detectives = this.$parent.getDetectives();
```

Comment:
you shouldn't need this, since you have access to all the detectives from the props itself: https://github.com/azharkhan/scotland-yard/blob/bdeb92066e2176c0f2e8f15029aa92670352cc25/src/components/Map.vue#L50